uber/models.py

```diff
@@ -1654,6 +1654,14 @@ class MPointsForCash(MagModel):
     when        = Column(UTCDateTime, default=lambda: datetime.now(UTC))
 
 
+class PromoCode(MagModel):
+    price           = Column(Integer, default=0)
+    uses            = Column(Integer, default=1)
+    code            = Column(UnicodeText)
+    users           = Column(UnicodeText, default="")
```

Comment:
I'm not wild about this kind of csv column as a pseudo foreign key.  We do something similar for a lot of our option checkbox/dropdown columns, but not for anything foreign-key-like.

A better approach would probably be to add a new nullable foreign key to the ``Attendee`` model which points to this model.  If that column is null then there's no discount applied.  If it points to a row in ``PromoCode`` then we do apply the discount.